algorithmic_efficiency/workloads/wmt/wmt_jax/workload.py

```diff
@@ -12,10 +12,11 @@
 import jax.numpy as jnp
 import numpy as np
 import optax
+import sacrebleu
```

Comment:
What was the reason for switching to `sacrebleu` again?

Reply:
Yeah so the function `corpus_bleu` which we use for calculating the score is taken from belu.py and belu.py is dependent on sacrebleu's 1.3.1 version : https://github.com/mjpost/sacrebleu/blob/v1.3.1/sacrebleu.py
Current verision of the package is at 2.4.3. 
Current version (1.3.1) and belu.py gives errors because of this; they've changed the location of some of the variables and modified the __init__.py

In either case we need to install sacrebelu, because belu.py still require to use sacrebelu 1.3.1 so why not just use the latest methods and call the function from there so instead of bleu_score = bleu.corpus_bleu(predictions, [references]).score

I am currently doing:
bleu_score = sacrebleu.corpus_bleu(predictions, [references]).score

Reply:
The bleu score calculation in the current version of `sacrebleu` is not equivalent to what we used before, hence our manual solution (see #264). So if we want to ensure that bleu scores are comparable across different versions of the benchmark, `sacrebleu` is an exception and should not be updated, i.e. it should stay pinned to 1.3.1.

Reply:
@init-22 could you check that pinning it to 1.3.1 does not break anything and revert the change to the bleu_score calls?

Reply:
yeah makes sense, will get back to you on this one

Reply:
@runame @priyakasimbeg 
Changed it back to 1.3.1, all checks are also passed.

Also, I remove the sacrebleu package dependency altogether in my local machine, I kept only the relevant part from sacrebleu v1.3.1 and ran the code and it worked too! keeping all global vars and functions as the original one, this way we dont need to install sacrebleu!

If you think, that's something we should do, I can raise another PR specific to that change once this branch is merged.

Reply:
Ok sounds good!